Sources/ReleaseServer/ReleaseServer.WebApi/Extensions/DeploymentMetaInfoExtension.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using ReleaseServer.WebApi.Config;
+
+namespace ReleaseServer.WebApi.Extensions
+{
+    public static class DeploymentMetaInfoExtension
+    {
+        public static bool IsValid(this DeploymentMetaInfoModel deploymentMetaInfoModel)
```

Comment:
I eliminated this function incl. the obsolete unit tests. 

Regarding the tip with `Nullable<DateTime>`: I tested deserializing a deployment.jso without a `ReleaseDate` field and with the `ReleaseDate` field wit an empty string value.
The deserializer recognized, that the `ReleaseDate` was not set.

Because of that, i didn't change this property to a `Nullable`.

Do you agree or did i misunderstand your intention?

Reply:
Where do I find the unit test for this?

Reply:
I manually tested it, if you want, i can add a unit test for this cases

Reply:
Yes, things like correct serialization, deserialization and detection of invalid data **must** be covered by respective unit tests.

In general:
In the case you write any test (like this one) for manual testing - just create a unit test for that as this usually does not take more effort. Otherwise this is a waste of work.

Reply:
Okay, i can add unit tests for this.

In my opinion, it didn't make much sense to test the functionality of `[Newtonsoft.Json.JsonRequired]` in an extra unit test, because will be already tested in their project.

Reply:
Yes, and an one of the invalid models is a missing `ReleaseDate` in the JSON that will be validated by `[JsonRequired]`.  

But no problem i will add a test with and empty 'DateTime' & a `DateTime` with an empty string:)

Reply:
I added the unit tests in the second PR, the JsonSerializable<T> class got implemented, see: https://github.com/Traeger-GmbH/release-server/pull/59/commits/0a85c5876b45b04962e6ef40537f1f95f48e3327

In both cases, the deserializer recognized, that the ReleaseDate was not set.